src/components/CategoryFilters.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div
+    class="bg-white text-xs gap-6 p-5 rounded-md shadow-sm mb-12 overflow-x-auto"
```

Comment:
Aqui nessa div a gente tem o container desse elemento. 
Segundo o figma o container tem `16px` de padding. Também não precisamos do `gap-6` nem `mb-12` (a margem entre outros elementos deve ir sempre fora do componente para facilitar o reuso. 

```suggestion
    class="bg-white text-xs p-4 rounded-md shadow-sm overflow-x-auto"
```

Reply:
Ok